cli/plugin/plugin.go

```diff
@@ -109,19 +238,27 @@ func (p *Plugin) Path() string {
 //       for arg in "$@"; do
 //         echo "$arg"
 //       done
-//       if ! timeout 1 ping -c1 remote.buildbuddy.io; then
+//       # Note: redirect ping stdout to stderr, to avoid adding ping output to
+//       # bazel args.
```

Comment:
@siggisim I ran into this bug when writing a test plugin -- I imagine this'll be a pain point for plugin authors with the stdout-based API (having to remember to pipe unwanted output to stderr).